conf/rubocop_without_rspec.yml

```diff
@@ -9,6 +9,15 @@ AllCops:
     - 'tmp/**/*'
     - 'vendor/**/*'
 
+Bundler/GemComment:
+  Enabled: true
+  OnlyFor:
+    - 'version_specifiers'
+    - 'source'
```

Comment:
Do we need comments for source? Won't the usually be for a repo with a bundled gem?

Reply:
Just trying to cover the several ways of changing a gem's source.

Reply:
Oops. I was confusing this with the `path` option. Seems like we'd only use `source` if there was a gem name conflict between one of our gems in gems.salsify.com and a different gem in rubygems.org and we wanted to pick the rubygems.org version... Definitely a scenario that would warrant a comment!